Countries_application/Countries_application/CountryList.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  CountryList.swift
+//  Countries_application
+//
+//  Created by Mateusz Papież on 09/12/2022.
+//
+
+import SwiftUI
+
+struct Country: Identifiable {
+    let id = UUID()
+    let flag_image: String
```

Comment:
I know that's the preference but pls use in this project camel case while naming everything.

[https://www.theserverside.com/answer/Camel-case-vs-snake-case-Whats-the-difference?amp=1](https://www.theserverside.com/answer/Camel-case-vs-snake-case-Whats-the-difference?amp=1)